README.md

```diff
@@ -78,6 +79,7 @@ export OKC_DB_USER=username
 export OKC_DB_PASSWORD=password
 export SMARTY_STREETS_AUTH_ID=smartystreetsauthid
 export SMARTY_STREETS_AUTH_TOKEN=smartystreetsauthtoken
+export GOOGLE_ANALYTICS=UA-39303796-11
```

Comment:
@BretFisher mentioned in #139 that It's okay we use the staging analytics key, just not the production one, and more importantly that it's actually okay if we have this in the readme so contributors can work without possible bugs.